lib/rake/eyes.rake

```diff
@@ -34,17 +34,27 @@ end
 
 def force_merge_eyes_baselines(branch, base)
   ensure_merge_util
-  RakeUtils.system_stream_output "#{BASE_MERGE_UTIL_CALL} -n Code.org -o -s #{branch} -t #{base}"
+  RakeUtils.system_stream_output "#{BASE_MERGE_UTIL_CALL} -n Code.org -o true -s #{branch} -t #{base}"
+end
+
+def copy_eyes_baselines(branch, base)
+  ensure_merge_util
+  RakeUtils.system_stream_output "#{BASE_MERGE_UTIL_CALL} -n Code.org -c true -s #{branch} -t #{base}"
```

Comment:
Took me a minute to see the difference between this method and the one above.

It may be worth commenting at the top of this section (maybe where you define MERGE_UTIL_PATH) with the URL of the documentation for the merge utility
http://support.applitools.com/customer/en/portal/articles/2142886-using-multiple-branches-

Or even inline the jar's help text:
```
╰○ java -jar ~/Desktop/applitools-merge.jar 
    Parameters 		 | 	 command 	 | 	 Is Required? 	 | 	 Default Value 
  ---------------------------------------------------------------------------------------------- 
    Post URL		 | 	 -url 		 | 	 yes 		 | 	 -
    Application Name 	 | 	 -n 		 | 	 yes 		 | 	 - 
    Source Branch 	 | 	 -s 		 | 	 yes 		 | 	 - 
    Target Branch 	 | 	 -t 		 | 	 no 		 | 	 default 
    Overwrite Changes 	 | 	 -o 		 | 	 no 		 | 	 false 
    Copy All 		 | 	 -c 		 | 	 no 		 | 	 false 
    Delete Source Branch | 	 -d 		 | 	 no 		 | 	 false 

```